drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb

```diff
@@ -278,8 +299,12 @@ def report_item_fields_from_enrollment(report_items:, enrollment:, report:) # ru
       annual_income_assessment = annual_assessment(enrollment, hoh.first_date_in_program)
       exit_income_assessment = enrollment.income_benefits_at_exit
 
+      report_item.total_monthly_income_at_entry = entry_income_assessment&.TotalMonthlyIncome
+      report_item.total_monthly_income_from_source_at_entry = entry_income_assessment&.all_sources_and_amounts&.sum { |_k, amount| amount || 0 }
```

Comment:
Do we need to check if any of these sources are blank?  Sometimes a missing value is indicative of a failure to collect the info rather than a `0`  (Not sure that's relevant here though.

Reply:
My interpretation of the check this is being used in was that it is only looking to see if the total monthly amount matched the sum from the sources, so nils would be the same as zero's. I could definitely be wrong and happy to check more into it.